src/components/buttons.js

```diff
@@ -55,7 +56,11 @@ export function createInputButtonPair(options = {}) {
 
   const button = document.createElement("button");
   button.className = buttonClass;
-  button.textContent = buttonText;
+  if (buttonChildElement) {
+    button.append(buttonChildElement);
+  } else {
+    button.textContent = buttonText;
+  }
```

Comment:
Made some assumptions here. Not assuming this is what you will want @sabjorn

Reply:
good enough for jazz